src/System.Net.Primitives/src/System/Net/CookieCollection.cs

```diff
@@ -24,7 +24,8 @@ internal enum Stamp
 
         private readonly ArrayList m_list = new ArrayList();
 
-        private DateTime m_timeStamp = DateTime.MinValue; // Do not rename (binary serialization)
+        private int m_version; // Do not rename (binary serialization)
+        private DateTime m_TimeStamp = DateTime.MinValue; // Do not rename (binary serialization)
         private bool m_has_other_versions; // Do not rename (binary serialization)
```

Comment:
Desktop has
```
[OptionalField]
bool m_IsReadOnly;
```
This means it's OK to deserialize to Desktop a stream without this field. But does it mean that it's going to work to deserialize to Core a stream that DOES have this field? The documentation doesn't say it is. Perhaps you need this optional field also.